crates/containerd-shim-wasmedge/src/instance.rs

```diff
@@ -45,11 +37,11 @@ static mut STDERR_FD: Option<RawFd> = None;
 
 static DEFAULT_CONTAINER_ROOT_DIR: &str = "/run/containerd/wasmedge";
 
-type ExitCode = (Mutex<Option<(u32, DateTime<Utc>)>>, Condvar);
+type ExitCode = Arc<(Mutex<Option<(u32, DateTime<Utc>)>>, Condvar)>;
```

Comment:
```suggestion
use containerd_shim_wasm::sandbox::instance::ExitCode;
```
(and merge with line 11)

Reply:
Done.